spec/enumerables_spec.rb

```diff
@@ -0,0 +1,129 @@
+require_relative '../enumerables.rb'
+
+describe Enumerable do
+  let(:array) { [1, 2, 3, 4, 5, 6, 7, 8, 9] }
+  let(:words_array) { %w[ant bear cat] }
+  let(:nil_true) { [nil, true, 99, false] }
+  let(:proced) { proc { |n| n * 2 } }
+  describe '#my_each' do
+    it 'Returns each element' do
+      expect(array.my_each { |n| n * n }).to eql(array)
+    end
+
+    it 'Returns enumerator' do
+      expect(array.my_each.class).to be(Enumerator)
+    end
+  end
```

Comment:
You need to add a more test to check that it actually can call the given block. Check out this one,
```ruby
it 'calls the given block for each of the items in the array' do
      each_output = 0
      block = proc { |num| each_output += num }
      array.my_each(&block)
      my_each_output = each_output.dup
      each_output = 0
      array.each(&block)
      expect(my_each_output).to eq(each_output)
 end

```

Reply:
Done! Thank you for pointing it out! We added the test case!